src/apps/explorer/pages/AppData/config.tsx

```diff
@@ -17,26 +17,21 @@ export const INITIAL_FORM_VALUES = {
   metadata: {},
 }
 
-export const INVALID_IPFS_CREDENTIALS = [
-  'Type error',
-  "Failed to execute 'setRequestHeader' on 'XMLHttpRequest': String contains non ISO-8859-1 code point.",
-]
+// export const INVALID_IPFS_CREDENTIALS = [
```

Comment:
Can we delete it at all?

Reply:
I would usually delete it, just kept it around since Anxo commented out the other part of the ipfs code when he dealt with this last.
It can always be found on git, so I'll remove it all.

Reply:
Actually, I think it should stay as it's not part of these changes.
I'll prepare a new PR to isolate them.

Reply:
https://github.com/cowprotocol/explorer/pull/672